.gitpod.yml

```diff
@@ -1,21 +1,19 @@
-# Using an updated Gitpod docker image (when using a non default JDK or tools)
-image:
-  file: .gitpod.Dockerfile
-
 tasks:
-  - init: sdk default java  11.0.15-tem && mvn clean verify
+  - name: Preview
+    command: gp ports await 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
+
+  - name: Run Jenkins
+    init: |
+      # Download and Install Java 17.0.6-tem
+      yes Y | sdk install java 17.0.6-tem
+
+    command: |
+      mvn hpi:run -Dhost=0.0.0.0      
 
 vscode:
   extensions:
-    - bierner.markdown-preview-github-styles
-    - github.vscode-pull-request-github
```

Comment:
Are these two extensions deleted because they are no longer available or because they are not useful enough?

Reply:
- The first extension previews what your markdown will look like rendered on Github.
- The second extension allows you to review and manage GitHub pull requests and issues in Visual Studio Code.

Both of these extensions are a matter of personal preferences and should not be part of a configuration which would be used by everyone